server/util/claims/BUILD

```diff
@@ -10,6 +10,7 @@ go_library(
         "//server/environment",
```

Comment:
uber nit (non blocking) - do we want these 2 pkgs under //enterprise?

Reply:
Good point, will move them there - but after all this to keep the diffs clean and not screw up my changes too much